src/tabs/tabs.scss

```diff
@@ -150,25 +167,39 @@
 }
 
 @mixin iui-tab-animated {
-  ~ .iui-tab-stripe {
-    height: $iui-xxs;
+  .iui-tab-stripe {
     position: absolute;
-    top: $iui-component-height - $iui-xxs;
-    @media (prefers-reduced-motion: no-preference) {
-      transition: width $iui-speed-fast ease-out, left $iui-speed-fast ease-out;
-    }
     @include themed {
       background-color: t(iui-color-foreground-primary);
     }
   }
 
-  &.iui-large ~ .iui-tab-stripe {
-    top: $iui-component-height-large - $iui-xxs;
-  }
-
-  &.iui-green ~ .iui-tab-stripe {
+  .iui-green ~ .iui-tab-stripe {
     @include themed {
       background-color: t(iui-color-foreground-positive);
     }
   }
+
+  &.iui-horizontal {
+    .iui-tab-stripe {
+      height: $iui-xxs;
+      top: $iui-component-height - $iui-xxs;
+      @media (prefers-reduced-motion: no-preference) {
+        transition: width $iui-speed-fast ease-out, left $iui-speed-fast ease-out;
+      }
+    }
+
+    .iui-large ~ .iui-tab-stripe {
+      top: $iui-component-height-large - $iui-xxs;
```

Comment:
@mayank99 Maybe we should also set this in JS like `left` for vertical tabs.

Reply:
What would be the benefit (other than consistency)? It doesn't change so it made sense to keep it in CSS

Reply:
I am not a fan of calculating stuff in CSS 😃 If one part changes, then another can break. But we can leave it as it is.